test/studies/shootout/submitted/knucleotide3.chpl

```diff
@@ -57,7 +57,7 @@ proc writeFreqs(data, param nclSize) {
   var arr = for (s,f) in freqs.items() do (f,s);
 
   // print the array, sorted by decreasing frequency
-  for (f, s) in arr.sorted(reverseComparator) do
+  for (f, s) in sorted(arr, reverseComparator) do
```

Comment:
Submitted shootouts should not be updated in their body until the code on the website has been updated, instead we update their .good files if we need to or mark them as .notest if they do not compile at all.  There's a README in this directory describing what to do

Reply:
Whoah, good to know; I'm glad I got this reviewed.  I've reverted the change in these two files and updated the corresponding `.good` file to include the new deprecation messages